test/components/prometheus-operator/prometheus_operator_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build packet
+// +build aws packet
```

Comment:
Also enable it for the Prometheus tests that we merged recently!

Reply:
Done.